src/prefect/cli/__init__.py

```diff
@@ -33,8 +33,12 @@ def make_user_config():
     with open(user_config_path, "w") as user_config:
         user_config.write(
             "# This is a user configuration file.\n"
-            "# Settings placed here will overwrite Prefect's defaults."
+            "# Settings placed here will overwrite Prefect's defaults.\n"
         )
+        config_loc = os.path.join(__file__, "../config.toml")
+        with open(config_loc) as defaults_file:
+            for line in defaults_file:
+                user_config.write()
```

Comment:
So this writes a blank line; you probably want either:
```python
user_config.write(defaults_file.read())
```
_or_
```python
for line in defaults_file.readlines():
    user_config.write(line)
```
it's basically personal preference as to which one you choose.